pkg/reconciler.v1/common/gang_scheduler_framework.go

```diff
@@ -38,6 +38,7 @@ type SchedulerFrameworkReconciler struct {
 	BaseGangReconciler
 	ReconcilerUtilInterface
 	client.Client
+	SchedulerName string
 }
 
 func BareSchedulerFrameworkReconciler(client client.Client, bgReconciler *BaseGangReconciler, enabled bool) *SchedulerFrameworkReconciler {
```

Comment:
What is this for @tenzen-y ?

Reply:
Maybe, this function is expected using in tests like the following:

https://github.com/kubeflow/common/blob/4dbf18c85417b54158914d1d01c1744b4e3542f2/test_job/reconciler.v1/test_job/test_job_reconciler.go#L55

Reply:
However, we don't have any test reconciler for scheduler-plugins for now.

Reply:
so it basically schedules the pods?

Reply:
Ah, I remind the purpose of this function.
This function isn't only for tests.

This just creates a new instance for the `SchedulerFrameworkReconciler`.

However, we use controllers instead of reconcilers. So we don't use this package for now.
More details:  #140

Reply:
> However, we use controllers instead of reconcilers. So we don't use this package for now. More details: #140
yes, we don't use this package for now.